doc/sphinx/conf.py.in

```diff
@@ -239,7 +239,9 @@ html_static_path = ['_static']
 #html_use_smartypants = True
 
 # Custom sidebar templates, maps document names to template names.
-#html_sidebars = {}
+# We set this to nothing otherwise in recent versions of the sphinx_bootstrap_theme
+# by default a sidebar could appear
+html_sidebars = {'**': []}
```

Comment:
Here (and below) could you perhaps put a reference from where you got this info? Might be useful if someone has to touch these things again.